src/providers/LocalVideoProvider/docs/LocalVideoProvider.stories.mdx

```diff
@@ -2,31 +2,23 @@
 
 # LocalVideoProvider
 
-The `LocalVideoProvider` provides video tile id associated with the local user's video, `isVideoEnabled` specifying the local video status, a function to set the value of `isVideoEnabled` and a function to toggle user's local video. 
-
-The React SDK depends on the Amazon Chime SDK for JavaScript, wherein, a single video stream is attached to a video element at a time. Hence, if `PreviewVideo` and `LocalVideo` component both are rendered on the same page and if both are enabled, stopping the video preview in `PreviewVideo` component will result into disconnecting the video stream for `LocalVideo` as well showing a black screen.
-
-To start/stop a video preview, please check `PreviewVideo` component [documentation](/docs/sdk-components-previewvideo--page).
-To start/stop user's local video, use `toggleVideo` function from the `LocalVideoProvider`.
+The `LocalVideoProvider` provides `tileId` of video tile associated with the local video, `isVideoEnabled` specifying the local video state, `hasReachedVideoLimit` specifying whether or not video limit is reached, and `toggleVideo()` to toggle local video.
 
+The state of the local video and the state of the previewVideo are independent.
```

Comment:
May be add a newline after this line. Also, suggest using `LocalVideo` over "local video" and `PreviewVideo` over "previewVideo".

Reply:
This is something I want to do as well, updated.